Unable to run tests

When I follow the instructions to run the tests, I get the following error:
```
./test
==================================================================== test session starts ====================================================================
platform linux -- Python 3.12.3, pytest-8.3.4, pluggy-1.5.0
rootdir: /home/zetatwo/Projects/genieutils-py
configfile: pyproject.toml
plugins: cov-6.0.0
collected 0 items / 2 errors                                                                                                                                
/home/zetatwo/.local/lib/python3.12/site-packages/coverage/control.py:892: CoverageWarning: No data was collected. (no-data-collected)
  self._warn("No data was collected.", slug="no-data-collected")

========================================================================== ERRORS ===========================================================================
___________________________________________________________ ERROR collecting tests/test_common.py ___________________________________________________________
ImportError while importing test module '/home/zetatwo/Projects/genieutils-py/tests/test_common.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python3.12/importlib/__init__.py:90: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/test_common.py:5: in <module>
    from genieutils.common import GenieClass, ByteHandler
E   ModuleNotFoundError: No module named 'genieutils'
_____________________________________________________ ERROR collecting tests/test_dat_compatibility.py ______________________________________________________
ImportError while importing test module '/home/zetatwo/Projects/genieutils-py/tests/test_dat_compatibility.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python3.12/importlib/__init__.py:90: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/test_dat_compatibility.py:6: in <module>
    from genieutils.common import ByteHandler
E   ModuleNotFoundError: No module named 'genieutils'

---------- coverage: platform linux, python 3.12.3-final-0 -----------
Coverage HTML written to dir htmlcov

================================================================== short test summary info ==================================================================
ERROR tests/test_common.py
ERROR tests/test_dat_compatibility.py
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 2 errors during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
===================================================================== 2 errors in 0.38s =====================================================================
All checks passed!
Success: no issues found in 19 source files

```

--- a/README.md
+++ b/README.md
@@ -59,7 +59,9 @@ data.save('path/to/modded/empires2_x2_p1.dat')
    `source venv/bin/activate`
 3. Install the dev dependencies  
    `pip install -r requirements-dev.txt`
-4. Run the test script  
+4. Install the local checkout of genieutils-py in editable mode
+   `pip install -e .`
+5. Run the test script
    `./test`
 
 

